t/run/todo.t

```diff
@@ -100,6 +100,21 @@ my $is_debugging_build = $Config{cppflags} =~ /-DDEBUGGING/;
 
 our $TODO;
 
+TODO: {
+    local $::TODO = 'GH 5835';
+    my $results = fresh_perl(<<~'EOF', {});
+        $^W = 1;
+        print "$^W";
```

Comment:
This `print` statement and the two subsequent ones may have been useful in preparing this pull request, but they don't actually print when the test is run and don't add anything to the logic of the test.  They should be removed.